components/service/glean/src/main/java/mozilla/components/service/glean/ping/PingMaker.kt

```diff
@@ -101,7 +79,7 @@ internal class PingMaker(
         // date the object was initialized.
         val startTime = if (pingName in pingStartTimes) pingStartTimes[pingName] else objectStartTime
         pingInfo.put("start_time", startTime)
-        val endTime = getISOTimeString()
+        val endTime = getISOTimeString(Date())
```

Comment:
Can we just add a default value for the input parameter here that defaults to `Date()`?